src/main/java/com/epam/reportportal/service/ReportPortal.java

```diff
@@ -191,17 +218,25 @@ public SaveLogRQ apply(String itemId) {
 
 				return rq;
 			}
-		});
+		};
 	}
 
 	public static boolean emitLog(final ReportPortalMessage message, final String level, final Date time) {
-		return emitLog(new com.google.common.base.Function<String, SaveLogRQ>() {
+		return emitLog(getLogSupplier(message, level, time));
+	}
+
+	public static boolean emitLaunchLog(final ReportPortalMessage message, final String level, final Date time) {
+		return emitLaunchLog(getLogSupplier(message, level, time));
```

Comment:
What logSupplier does it use? 
As I see it uses the same as for test item. So you provide item id for it. Is it expected?